common/supplemental/supplementalData.xml

```diff
@@ -4402,342 +4402,344 @@ XXX Code for transations where no currency is involved
           </eras>
         </calendar>
         <calendar type="islamic-tbla">
           <calendarSystem type="lunar" />
           <eras>
-              <era type="0" start="622-7-15"/>
+              <era type="0" start="622-7-15" code="islamic-tbla" aliases="ah"/>
           </eras>
         </calendar>
         <calendar type="islamic-umalqura">
           <calendarSystem type="lunar" />
           <eras>
-              <era type="0" start="622-7-15"/>
+              <era type="0" start="622-7-15" code="islamic-umalqura" aliases="ah"/>
           </eras>
         </calendar>
         <calendar type="chinese">
           <calendarSystem type="lunisolar"/>
           <eras>
-              <era type="0" start="-2636-01-01"/> <!-- 'sequential' year -->
+              <era type="0" start="-2636-01-01" code="chinese"/> <!-- 'sequential' year -->
           </eras>
         </calendar>
         <calendar type="hebrew">
           <calendarSystem type="lunisolar"/>
           <eras>
-              <era type="0" start="-3760-10-7"/>
+              <era type="0" start="-3760-10-7" code="hebrew" aliases="am"/>
           </eras>
         </calendar>
         <calendar type="buddhist">
           <calendarSystem type="solar" />
           <eras>
-              <era type="0" start="-542-01-01"/>
+              <era type="0" start="-542-01-01" code="buddhist" aliases="be"/>
           </eras>
         </calendar>
         <calendar type="coptic">
           <calendarSystem type="other"/>
           <eras>
-              <era type="0" end="284-08-28"/>
-              <era type="1" start="284-08-29"/>
+              <era type="0" end="284-08-28" code="pre-coptic"/>
+              <era type="1" start="284-08-29" code="coptic"/>
           </eras>
         </calendar>
         <calendar type="persian">
           <calendarSystem type="solar"/>
           <eras>
-              <era type="0" start="622-01-01"/>
+              <era type="0" start="622-01-01" code="persian" aliases="ap"/>
           </eras>
         </calendar>
         <calendar type="dangi">
           <calendarSystem type="lunisolar"/>
           <eras>
-              <era type="0" start="-2332-01-01"/> <!-- 'sequential' year -->
+              <era type="0" start="-2332-01-01" code="dangi"/> <!-- 'sequential' year -->
           </eras>
         </calendar>
         <calendar type="ethiopic">
           <calendarSystem type="other"/>
           <eras>
-              <era type="0" end="8-08-28"/>
-              <era type="1" start="8-08-29"/>
+              <era type="0" end="8-08-28" code="ethioaa" aliases="ethiopic-amete-alem mundi"/>
+              <era type="1" start="8-08-29" code="ethiopic" aliases="incar"/>
           </eras>
         </calendar>
         <calendar type="ethiopic-amete-alem">
           <eras>
-              <era type="0" end="-5492-08-29"/>
+              <era type="0" end="-5492-08-29" code="ethioaa" aliases="ethiopic-amete-alem mundi"/>
```

Comment:
the type in this calendar is "ethiopic-amete-alem" should the code in here be also "ethiopic-amete-alem" and the aliases be "ethioaa mundi" instead?

Reply:
I understand this now.